src/main/java/htsjdk/samtools/CRAMBAIIndexer.java

```diff
@@ -296,8 +296,11 @@ public static void createIndex(final SeekableStream stream,
      * One instance is used to construct an entire index.
      * processAlignment is called for each alignment until a new reference is encountered, then
      * processReference is called when all records for the reference have been processed.
+     *
+     * TODO: this was copied from {@link htsjdk.samtools.BAMIndexer.BAMIndexBuilder}

```

Comment:
This link is broken because this class is not accessible from here; I think either just remove the `@link` or replace with `@see`.